templates/docs/docs/Makefile.j2

```diff
@@ -2,11 +2,13 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -n
-SPHINXBUILD   = sphinx-build
-PAPER         =
-BUILDDIR      = _build
-PULP_URL      = "http://localhost:24817"
+SPHINXOPTS    		  = -W -n
+SPHINXBUILD   		  = sphinx-build
+PAPER         		  =
+BUILDDIR      		  = _build
+DIAGRAM_BUILD_DIR     = _diagrams
+DIAGRAM_SOURCE_DIR    = diagrams_src
+PULP_URL      		  = "http://localhost:24817"
```

Comment:
Somehow, the alignment of the equals went wrong here? Is this some uncommon tab expansion?
I would not even bother to align them, as in "Why do i need to touch all the lines if one variable name becomes longer?"

Reply:
this was looking different on my editor, I think it was a space/tab issue with my editor config

Reply:
I checked it at vscode, sublime, and vim. And I couldn't reproduce it locally
![image](https://user-images.githubusercontent.com/17153022/93780671-d5798f80-fbfe-11ea-8b96-ef411a76df04.png)